README.md

```diff
@@ -108,9 +108,8 @@ Show some ❤️&nbsp; by giving <img src="https://imgur.com/o7ncZFp.jpg" height
  # Part of Open Source Program
```

Comment:
Please include  all that  label  and logo section  after the cover image  respectively . 
Thank u :smile: